packages/framework/block-std/src/gfx/surface/element-model.ts

```diff
@@ -134,6 +144,10 @@ export abstract class GfxPrimitiveElementModel<
     return true;
   }
 
+  get container() {
```

Comment:
What's the difference between group and container?

Reply:
The contianer() can return block contianer like frame